portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +24,23 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  // Regex that accepts letters, numbers, and spaces
+  private static final String regex = "^[A-Za-z0-9 ]*$";
```

Comment:
Could you rename this to something more descriptive of what the regex represents? Additionally, Google's Java style requires constants are in UPPER_SNAKE_CASE. Example solutions: "ALPHANUMERIC" or "ALPHANUMERIC_WITH_SPACES".
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
Oh I didn't know that there was a consolidated Java style guide for Google, thanks for the link! I've renamed the const variable.